docs/faststore/components/features/auth-content.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: 'Authenticated content'
+---
```

Comment:
FastStore docs don't need any other info in the frontmatter? E.g. url, etc

Reply:
No need @julia-rabello. The slug, for example, is defined by the filename.